--- src/routes/index.ts
+++ src/routes/index.ts
@@ -10,6 +10,7 @@
 import { setPasswordRoutes } from './setPassword';
 import { sessionDetailScheduleRouter } from './sessionDetailSchedule';
 import participantImportRoutes from './participantImport';
+import pesertaRoutes from './peserta';
 
 const routes = (app: Application): void => {
   const router = Router(); // this value is null at the beginning
@@ -35,6 +36,7 @@
   mediaRoutes(router);
   sessionDetailScheduleRouter(router);
   participantImportRoutes(router);
+  pesertaRoutes(router);
 };
 
 export default routes;
