--- src/routes/schedule.ts
+++ src/routes/schedule.ts
@@ -5,6 +5,7 @@
 import * as scheduleValidationMiddlewares from "../middlewares/validation/schedule";
 import * as commonValidationMiddleware from "../middlewares/validation/common";
 import * as scheduleMiddlewares from "../middlewares/schedule";
+import * as participantController from "../controllers/participant";
 
 export const scheduleRoutes = (router: Router): void => {
     const scheduleRouter = Router();
@@ -35,6 +36,15 @@
         scheduleController.getAllSchedules,
     )
 
+    scheduleRouter.get(
+        '/:id/participant',
+        authMiddlewares.isAuthorized,
+        aclMiddlewares.isAdmin,
+        commonValidationMiddleware.validateIdParams,
+        scheduleMiddlewares.checkScheduleIdExists,
+        participantController.getAllParticipant,
+    )
+
     scheduleRouter.put(
         '/:id',
         authMiddlewares.isAuthorized,
